dgl_sparse/src/sparse_matrix.cc

```diff
@@ -0,0 +1,32 @@
+/*!
+ *  Copyright (c) 2022 by Contributors
+ * \file sparse/sparse_matrix.cc
+ * \brief DGL C++ sparse matrix implementations
+ */
+#include <sparse/elementwise_op.h>
+#include <sparse/sparse_matrix.h>
+
+namespace dgl {
+namespace sparse {
+
+std::vector<torch::Tensor> SparseMatrix::COOTensors() {
+  auto coo = COOPtr();
+  auto val = Value();
+  return {coo->row, coo->col, val};
```

Comment:
No need to return value tensor.

Reply:
We may need to return value to make it consist with `CSRTensors()` and `CSCTensors()`.

Reply:
Let's discuss this offline. Won't hold the PR for this.